nc-news-react/src/components/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react"
+import { getArticles } from "../axios"
+import { Link } from "react-router-dom"
+
+function Articles () {
+
+   const [articles, setArticles] = useState([])
+   const [isLoading, setisLoading] =useState(true)
+   const [isErr, setisErr] = useState(false)
+
+    useEffect(()=> {
+        getArticles().then((articlesData)=> {
+            setArticles(articlesData)
+            setisLoading(false)
+        }).catch((err)=> setisErr(true) )
+    }, [])
+
+    console.log(articles)
+
+if (isLoading === true){return <p>Loading...</p>}
+
+    return <div className="articlesSection" >
+        <h1>Articles</h1>
```

Comment:
For accessibility, readability and navigability, it is bad practice to have more than one h1 on the page at any given time - you've got one here and one in your header too. If you want the font to appear bigger without using an H1, you can handle this in your styling.